Amazon.QLDB.Driver/client/Session.cs

```diff
@@ -272,11 +273,30 @@ internal virtual CommitTransactionResult CommitTransaction(string txnId, MemoryS
                     return valueHolder;
                 });
 
+                return await this.ExecuteStatementAsync(txnId, statement, valueHolders.ToArray(), cancellationToken);
+            }
+            finally
```

Comment:
Why was this moved? This ExecuteStatementAsync seems to call the other, so wouldn't it make more sense for the finally check to be in the other?

Reply:
Reverted the finally block to the base function